packages/stryker/src/ScoreResultCalculator.ts

```diff
@@ -0,0 +1,125 @@
+import { MutantResult, MutantStatus, ScoreResult } from 'stryker-api/report';
+import { freezeRecursively } from './utils/objectUtils';
+import * as _ from 'lodash';
+import * as path from 'path';
+
+export default class ScoreResultCalculator {
+
+  static calculate(results: MutantResult[], basePath: string = ''): ScoreResult {
+    const name = this.determineCommonBasePath(results, basePath);
+    const childResults = this.calculateChildScores(results, name, basePath);
+    const numbers = this.calculateNumbers(results, childResults);
+    return freezeRecursively(_.assign(numbers, { name, childResults }));
+  }
+
+  private static compareScoreResults(a: ScoreResult, b: ScoreResult) {
+    const sortValue = (scoreResult: ScoreResult) => {
+      // Directories first
+      if (scoreResult.childResults.length) {
```

Comment:
We won't fix this until we have to know the difference between a file and a directory in a reporter.